src/test/java/samples/android/nativeapp/apidemos/screens/ViewControlsScreen.java

```diff
@@ -1,25 +1,52 @@
 package samples.android.nativeapp.apidemos.screens;
 
+import aquality.appium.mobile.actions.SwipeDirection;
+import aquality.appium.mobile.elements.actions.ElementTouchActions;
+import aquality.appium.mobile.elements.interfaces.IButton;
 import aquality.appium.mobile.elements.interfaces.ICheckBox;
+import aquality.appium.mobile.elements.interfaces.ILabel;
 import aquality.appium.mobile.elements.interfaces.IRadioButton;
 import aquality.appium.mobile.screens.AndroidScreen;
 import io.appium.java_client.MobileBy;
 import org.openqa.selenium.By;
 
 public class ViewControlsScreen extends AndroidScreen {
-    public ViewControlsScreen() {
-        super(By.id("android:id/content"), "View/Controls");
-    }
 
-    public IRadioButton getRadioButton(int number){
+    public IRadioButton getRadioButton(int number) {
         return getElementFactory().getRadioButton(
                 MobileBy.AccessibilityId(String.format("RadioButton %d", number)),
                 String.valueOf(number));
     }
 
-    public ICheckBox getCheckBox(int number){
+    public ICheckBox getCheckBox(int number) {
         return getElementFactory().getCheckBox(
                 MobileBy.AccessibilityId(String.format("Checkbox %d", number)),
                 String.valueOf(number));
     }
+
+    private final IButton btnDisabled = getElementFactory().getButton(MobileBy.id("button_disabled"), "Disabled");
+
+    public ViewControlsScreen() {
+        super(By.id("android:id/content"), "View/Controls");
+    }
+
+    public void scrollToAllInsideScrollViewLabel() {
+        new ElementTouchActions().scrollToElement(lblAllInsideScrollView, SwipeDirection.DOWN);
```

Comment:
do we check a Left/Right scroll in tests?

Reply:
Not yet. Will try to find the activity in the test app, where we can check horizontal scrolls as well.